docs/docs/how-to/cloud/create-site-from-repository.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Create a Site from a Repository"
+description: "How to create a Gatsby Cloud site from a repository"
```

Comment:
Nit-picking: the description format in these docs are not consistent. Can we try to sync them up with "Learn how to..."?

```suggestion
description: "Learn how to create a Gatsby Cloud site from a repository"
```